spec/features/invoices/show_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe 'invoices show page', js: true do
+  include MoneyHelper
+
+  let(:user) { create :user }
+  let(:org)  { create :organization, with_user: user }
+
+  before do
+    sign_in user
+  end
+
+  subject{ page }
+
+  context 'Download as PDF' do
+    let!(:invoice) { create :invoice, organization: org }
+
+    context 'Download' do
```

Comment:
@yunixon I'm not sure that this context named correctly. Content should be named in readable format, so when you read spec - it should be readable like sequence. Please try to use something like `with usual download mode`.